platforms/snips-nlu-ontology-kotlin/src/main/kotlin/ai/snips/nlu/ontology/Ontology.kt

```diff
@@ -1,14 +1,6 @@
 package ai.snips.nlu.ontology
 
-import ai.snips.nlu.ontology.SlotValue.Type.AMOUNT_OF_MONEY
-import ai.snips.nlu.ontology.SlotValue.Type.CUSTOM
-import ai.snips.nlu.ontology.SlotValue.Type.DURATION
-import ai.snips.nlu.ontology.SlotValue.Type.INSTANT_TIME
-import ai.snips.nlu.ontology.SlotValue.Type.NUMBER
-import ai.snips.nlu.ontology.SlotValue.Type.ORDINAL
-import ai.snips.nlu.ontology.SlotValue.Type.TEMPERATURE
-import ai.snips.nlu.ontology.SlotValue.Type.TIME_INTERVAL
-import ai.snips.nlu.ontology.SlotValue.Type.PERCENTAGE
+import ai.snips.nlu.ontology.SlotValue.Type.*
```

Comment:
avoid .* imports (there is an option in intellij to change the number at witch it starts *ring

Reply:
Ok